caliban_toolbox/dataset_splitter_test.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2016-2020 The Van Valen Lab at the California Institute of
+# Technology (Caltech), with support from the Paul Allen Family Foundation,
+# Google, & National Institutes of Health (NIH) under Grant U24CA224309-01.
+# All rights reserved.
+#
+# Licensed under a modified Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.github.com/vanvalenlab/caliban-toolbox/LICENSE
+#
+# The Work provided may be used for non-commercial academic purposes only.
+# For any other use of the Work, including commercial use, please contact:
+# vanvalenlab@gmail.com
+#
+# Neither the name of Caltech nor the names of its contributors may be used
+# to endorse or promote products derived from this software without specific
+# prior written permission.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import pytest
+
+import numpy as np
+
+from caliban_toolbox.dataset_splitter import DatasetSplitter
+
+
+def test__init__():
+    seed = 123,
+    splits = [0.5, 0.75, 1]
+
+    ds = DatasetSplitter(seed=seed, splits=splits)
+
+    assert ds.seed == seed
+    assert ds.splits == splits
+
+    # unsorted splits get sorted
+    splits = [0.8, 0.3, 0.5]
+    ds = DatasetSplitter(seed=seed, splits=splits)
+    splits.sort()
+
+    assert ds.splits == splits
```

Comment:
`splits` is the same list as `ds.splits`, and this test will pass if you comment out the `splits.sort()`. Instead, maybe you can make an assertion about the elements of the list.